cmd/system-probe/app/debug.go

```diff
@@ -39,7 +39,7 @@ func debugRuntime(_ *cobra.Command, args []string) error {
 	}
 
 	// TODO rather than allowing arbitrary query params, use cobra flags
-	r, err := util.DoGet(c, "http://localhost/debug/"+args[0])
+	r, err := util.DoGet(c, "http://localhost/debug/"+args[0], util.LeaveConnectionOpen)
```

Comment:
I think this could be `util.CloseConnection`, since the process will exit after this request is made.

Reply:
Sure! I've just updated it. 
I ended up using `util.LeaveConnectionOpen` in all places except the ones where we were leaking connections in core and process agents to avoid changing multiple behaviors during QA.